packages/manifest/src/index.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TeamsAppManifest } from "./manifest";
+import fs from "fs-extra";
+import Ajv from "ajv-draft-04";
+import { JSONSchemaType } from "ajv"; 
+import axios, { AxiosResponse } from "axios";
+
+export * from "./manifest";
+export type TeamsAppManifestJSONSchema = JSONSchemaType<TeamsAppManifest>;
+
+/**
+ * Loads the manifest from the given path without validating its schema.
+ * 
+ * @param path - The path to the manifest file.
+ * @throws Will propagate any error thrown by the fs-extra#readJson.
+ * 
+ * @returns The Manifest Object. 
+ */
+export async function loadFromPath(path: string): Promise<TeamsAppManifest> {
+  return fs.readJson(path);
+}
+
+/**
+ * Writes the manifest object to the given path.
+ * 
+ * @param path - Where to write
+ * @param manifest - Manifest object to be saved
+ * @throws Will propagate any error thrown by the fs-extra#writeJson.
+ * 
+ */
+export async function writeToPath(path: string, manifest: TeamsAppManifest): Promise<void> {
+  return fs.writeJson(path, manifest)
```

Comment:
Can we add: `JSON.stringify(manifest, null, 4)` or something similar? Otherwise, the file created is messy

Reply:
Sure, will do.